makefile

```diff
@@ -44,4 +44,4 @@ generate-end-to-end:
 
 generate:
 	statik -f -src=./docs/migrations -dest=./internal/migrations
```

Comment:
it would be nice for this to work independently out-of-the-box, can you add a `build-deps` target that runs a go get for statik and kafmesh-gen?